string[] name
float64[] position
float64[] velocity
float64[] effort
float64[] position_command
float64[] velocity_command
float64[] effort_command
geometry_msgs/Vector3[] accelerometer
geometry_msgs/Vector3[] gyro
float64[] deflection
float64[] deflection_velocity
float64[] motor_velocity
float64[] motor_current
float64[] motor_winding_current
float64[] motor_sensor_temperature
float64[] motor_winding_temperature
float64[] motor_housing_temperature
float64[] board_temperature
float64[] processor_temperature
float64[] voltage
std_msgs/ColorRGBA[] led_color
uint64[] sequence_number
uint64[] receive_time
uint64[] transmit_time
uint64[] hardware_receive_time
uint64[] hardware_transmit_time

